Sources/MySQL/MySQL+Date.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+
+public enum MySQLDateError: Swift.Error {
+    case invalidDate
+}
+
+extension Date {
+    public init(mysql date: String) throws {
+        guard let date = dateFormatter.date(from: date) else {
+            throw MySQLDateError.invalidDate
+        }
+        
+        self = date
+    }
+    
+    public var mysql: String {
+        return dateFormatter.string(from: self)
+    }
+}
+
+private var _df: DateFormatter?
```

Comment:
This date formatter code needs at least a little explaining probably. If i saw this i would try to figure out why it's there.

Maybe something like `// date formatter init is slow, need to reuse`